utils/taskcluster_downloader.py

```diff
@@ -64,14 +66,29 @@ def donwload_logs(group_id, output):
         os.makedirs(output, exist_ok=True)
         output_path = os.path.join(output, f"{task_obj_label}.log")
 
-        print(f"Downloading {output_path}")
-        with open(output_path, "wb") as f:
-            downloadArtifactToFile(
-                f,
-                taskId=task["status"]["taskId"],
-                name="public/build/train.log",
-                queueService=queue,
-            )
+        url = queue.buildUrl("getLatestArtifact", task_id, "public/logs/live.log")
+        resp = requests.get(url, stream=True, timeout=5)
+        print(f"Downloading {url}")
+        lines = []
```

Comment:
suggestion (non-blocking): consider not saving all of the lines in memory. AFAICT neither `lines` nor `log_lines` are necessary - I believe you could open the output file immediately, and put the `start_writing` logic in that loop instead.